api/pkg/db/model/model.go

```diff
@@ -21,33 +21,32 @@ import (
 type (
 	Category struct {
 		gorm.Model
-		Name string `gorm:"size:100;not null;unique"`
+		Name string `gorm:"not null"`
 		Tags []Tag
 	}
 
 	Tag struct {
 		gorm.Model
-		Name       string `gorm:"size:100;not null;unique"`
+		Name       string `gorm:"not null"`
```

Comment:
Similarly here also there can't be 2 tags with same name. So, unique can be applied here.